contracts/external/barnbridge/BarnInterface.sol

```diff
@@ -0,0 +1,13 @@
+pragma solidity >=0.6.0 <0.7.0;
+
+import "@openzeppelin/contracts-upgradeable/token/ERC20/IERC20Upgradeable.sol";
+
+interface BarnInterface is IERC20Upgradeable {
```

Comment:
Can you point me to the Barn staking contract that you have used in order to find these functions?

Reply:
@Perseverance As far as I understood, this is it: https://github.com/LimeChain/BarnBridge-Barn/blob/master/contracts/facets/BarnFacet.sol